helm-chart/images/binderhub/Dockerfile

```diff
@@ -1,8 +1,8 @@
 # Using multi-stage builds
-ARG DIST=buster
+ARG DIST=slim-buster
```

Comment:
I'm not sure buildpack-deps has `slim` variants (there are kind un-slim by definition). So maybe need DIST to be `buster` and add `slim` below in `python:3.8-slim-$DIST`?